folly/Portability.h

```diff
@@ -284,6 +284,17 @@ inline size_t malloc_usable_size(void* ptr) {
 # define FOLLY_HAS_RTTI 1
 #endif
 
+#if defined(_MSC_VER) && _MSC_FULL_VER <= 190023026 // 2015 RTM or below
```

Comment:
How concerned do we need to be about checking for which pre-relesae we're working with?
Seems like we can just assume "MSVC 2015 Latest" until final comes out.

Can we kill this check and assume the two defines below are just not set?
Makes the code a lot simpler if so.